bot/exts/evergreen/catify.py

```diff
@@ -0,0 +1,44 @@
+import random
+from typing import Optional
+
+from discord.ext import commands
+
+from bot.constants import Cats
+
+
+class Catify(commands.Cog):
+    """Cog for the catify command."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @commands.command(aliases=["ᓚᘏᗢify", "ᓚᘏᗢ"])
+    async def catify(self, ctx: commands.Context, *string: Optional[str]) -> None:
+        """Catifies your nickname or a given string."""
+        if string == ():
+            username = ctx.author.name
+            if len(username) >= 28:
+                await ctx.send("Your username is too long to be catified! Please change it.")
+            else:
+                username += f" | {random.choice(Cats.cats)}"
+                await ctx.send(f"Your catified username is: `{username}`")
+                await ctx.author.edit(nick=username)
+        else:
+            string = " ".join(string)
+            string_list = string.split()
+            for index, name in enumerate(string_list):
+                if "cat" in name:
+                    string_list[index] = string_list[index].replace("cat", random.choice(Cats.cats))
```

Comment:
```suggestion
                    string_list[index] = name.replace("cat", random.choice(Cats.cats))
```

Reply:
this breaks the command

Reply:
How so?

Reply:
throws an error

Reply:
What error are you getting? I don't get any error with that change.

Reply:
implemented